github/query.go

```diff
@@ -0,0 +1,112 @@
+package github
+
+import (
+	"encoding/json"
+
+	"github.com/azaky/graphb"
```

Comment:
I'm wondering if we can avoid pulling in an extra dependency to construct the GraphQL query?

After all, the only thing we need to do is POST some JSON to the `graphql` endpoint. The JSON payload can be constructed like this:

```go
variables := map[string]interface{}{
  "state": "open",
}
payload := map[string]interface{}{
  "variables": variables,
  "query": `
query(...) {
  repository(...) {
    pullRequests(...) {
      ...
    }
  }
}`,
}

// later:
api.PostJSONPreview("/graphql", payload, checksType)
```

I think it would be even simpler and more readable to write the query string by hand like that instead of using the library. In the future, when maintaining GraphQL queries becomes numerous and more cumbersome, we can consider using a library.

Reply:
Sure, we can :grin: . I just thought that there will be more GraphQL queries than only this one and it will be bloated for other developers if we are all using plain string but let's tackle that later. I am going to drop this dependency.